### ISSUE TITLE
Extension not working in python 3.8 (conda install python==3.8)

### ISSUE THREAD

**Issue by 9-2-1** [microsoft/vscode-mypy#365]
## Diagnostic Data
- Python version (& distribution if applicable, e.g., Anaconda): **conda 25.3.1, python 3.8**
- Type of virtual environment used (e.g., conda, venv, virtualenv, etc.): **conda**
- Operating system (and version): **Microsoft Windows [版本 10.0.22631.5262]** 
- Version of tool extension you are using: **2025.2.0**

### Expected Behavior

Type errors are detected.

### Actual Behavior

Nothing is detected even running 'mypy .' produce outputs.  
Found error in extension outputs.

## Reproduction Steps:

`conda install python==3.8`, then select the python interpreter.

## Logs:

<details>
  <summary>Click here for detailed logs</summary>
  

```
2025-04-27 21:07:07.415 [info] [Trace - 21:07:07] Received notification 'window/logMessage'.
2025-04-27 21:07:07.415 [info] c:\Users\11951\miniforge3\envs\tggw\python.exe -m mypy --version
2025-04-27 21:07:07.416 [info] [Trace - 21:07:07] Received notification 'window/logMessage'.
2025-04-27 21:07:07.416 [info] CWD Server: D:\Documents\常用\tggwpy
2025-04-27 21:07:07.632 [info] [Trace - 21:07:07] Received notification 'window/logMessage'.
2025-04-27 21:07:07.633 [info] Traceback (most recent call last):
  File "c:\Users\11951\miniforge3\envs\tggw\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\Users\11951\miniforge3\envs\tggw\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\__main__.py", line 9, in <module>
    from mypy.main import main, process_options
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\main.py", line 16, in <module>
    from mypy import build, defaults, state, util
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\build.py", line 42, in <module>
    import mypy.semanal_main
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\semanal_main.py", line 35, in <module>
    from mypy.checker import FineGrainedDeferredNode
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\checker.py", line 12, in <module>
    import mypy.checkexpr
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\checkexpr.py", line 16, in <module>
    from mypy import applytype, erasetype, join, message_registry, nodes, operators, types
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\applytype.py", line 6, in <module>
    import mypy.subtypes
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\subtypes.py", line 9, in <module>
    import mypy.constraints
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\constraints.py", line 10, in <module>
    import mypy.typeops
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\typeops.py", line 14, in <module>
    from mypy.copytype import copy_type
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\copytype.py", line 5, in <module>
    from mypy.types import (
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\types.py", line 22, in <module>
    import mypy.nodes
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\nodes.py", line 15, in <module>
    import mypy.strconv
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\strconv.py", line 11, in <module>
    from mypy.options import Options
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\options.py", line 13, in <module>
    from mypy.util import get_class_descriptors, replace_object_state
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\libs\mypy\util.py", line 33, in <module>
    TYPESHED_DIR: Final = str(importlib_resources.files("mypy") / "typeshed")
AttributeError: module 'importlib.resources' has no attribute 'files'

2025-04-27 21:07:07.633 [info] [Trace - 21:07:07] Received notification 'window/logMessage'.
2025-04-27 21:07:07.633 [info] 


2025-04-27 21:07:07.633 [info] [Trace - 21:07:07] Received notification 'window/logMessage'.
2025-04-27 21:07:07.633 [info] Version info for linter running for D:\Documents\常用\tggwpy:

2025-04-27 21:07:07.633 [info] [Trace - 21:07:07] Received notification 'window/logMessage'.
2025-04-27 21:07:07.633 [info] Error while checking mypy executable:
Traceback (most recent call last):
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\tool\lsp_server.py", line 123, in get_mypy_info
    first_line = result.stdout.splitlines(keepends=False)[0]
IndexError: list index out of range

2025-04-27 21:07:07.634 [info] [Trace - 21:07:07] Received notification 'window/logMessage'.
2025-04-27 21:07:07.634 [info] [Error - 21:07:07] Linting failed with error:
Traceback (most recent call last):
  File "c:\Users\11951\.vscode\extensions\ms-python.mypy-type-checker-2025.2.0\bundled\tool\lsp_server.py", line 234, in _linting_helper
    version = get_mypy_info(settings).version
AttributeError: 'NoneType' object has no attribute 'version'
```

</details>

## Outcome When Attempting Debugging Steps:

Did running it from the command line work? **Yes.**

## Extra Details

Python==3.8 fail but Python==3.12 works normally

**Comment by karthiknadig** [comment]
This is expected, Python 3.8 is deprecated. You can use the extension view to install the older version of mypy extension to make this work with Python 3.8
